pkgs/os-specific/linux/kernel/hardened/patches.json

```diff
@@ -1,4 +1,14 @@
 {
+    "4.14": {
```

Comment:
4.14 got revived here, not sure if we care?

Reply:
Dropped it again.